wagtail/admin/views/reports/aging_pages.py

```diff
@@ -68,7 +68,10 @@ def decorate_paginated_queryset(self, queryset):
         }
         for page in queryset:
             if page.last_published_by:
-                page.last_published_by_user = username_mapping[page.last_published_by]
+                page.last_published_by_user = username_mapping.get(
+                    page.last_published_by,
+                    f"user {page.last_published_by} (deleted)",
```

Comment:
It would be nice if this uses gettext, so it can be translated. Actually, if you use the same string as https://github.com/wagtail/wagtail/blob/main/wagtail/models/audit_log.py#L227, no further translation string gets created.
So I would suggest something like:
```python
                page.last_published_by_user = username_mapping.get(
                    page.last_published_by,
                )
                if page.last_published_by_user is None:
                    page.last_published_by_user = _("user %(id)s (deleted)") % {
                        "id": page.last_published_by,
                    }
```